tests/load/network_statistics_smoke_test.js

```diff
@@ -0,0 +1,25 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export const options = {
+  stages: [
+    { duration: '1m', target: 10 },
+    { duration: '5m', target: 20 },
+    { duration: '1m', target: 0 },
+  ],
+  thresholds: {
+    http_req_failed: ['rate<0.01'], // http errors should be less than 1%
+    http_req_duration: ['p(95)<200'], // 95% of requests should be below 200ms
+  },
+  noConnectionReuse: true,
+  userAgent: 'ExplorerLoadTest/1.0',
+};
+
+export default function () {
+  const hostUrl = __ENV.HOST_URL;
+  const defaulHostUrl = 'http://explorer-service:3002/dev';
```

Comment:
What is this url? It's the one you get when running the service locally?

How to run it locally?

Reply:
Maybe get the url from args, defaulting to localhost:3002

Reply:
> How to run it locally?

There is this [`README.md`](https://github.com/HathorNetwork/hathor-explorer-service/blob/e45f58f450f9fa6c75a080ec16fd3cde823677c1/tests/load/README.md) file in the `tests/load` folder.

Reply:
> Maybe get the url from args, defaulting to localhost:3002

Done: 5bae2b0b21b2a7d7fe09ca23dd009b61c0a4b881